flash/vision/classification/data.py

```diff
@@ -527,3 +534,46 @@ def from_filepaths(
             seed=seed,
             **kwargs
         )
+
+
+class _CustomBaseVisualization(BaseVisualization):
+    """Process and show the image batch and its associated label.
+    """
+    max_cols: int = 4  # maximum number of columns we accept
+
+    def show_per_batch_transform(self, batch: List[Any], running_stage):
```

Comment:
Where are the other hooks ?

Reply:
do we want to pop up 5 windows to the user ?